static/app/components/group/sidebar.tsx

```diff
@@ -178,12 +178,19 @@ class BaseGroupSidebar extends React.Component<Props, State> {
   }
 
   render() {
-    const {className, event, group, organization, project, environments} = this.props;
+    const {event, group, organization, project, environments} = this.props;
     const {allEnvironmentsGroupData, currentRelease, tagsWithTopValues} = this.state;
     const projectId = project.slug;
 
+    const hasPageFilters = organization.features.includes('selection-filters-v2');
+
     return (
-      <div className={className}>
+      <Container>
+        {hasPageFilters && (
+          <PageFiltersContainer>
+            <EnvironmentPageFilter alignDropdown="right" />
```

Comment:
should we just style `EnvironmentPageFilter` to have margin bottom instead of wrapping?

Reply:
We actually don't pass down `className`. I can update though. good call

Reply:
I'll update in a separate PR